sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -9,7 +9,6 @@
 #include "azure/core/http/pipeline.hpp"
 #include "azure/core/nullable.hpp"
 #include "azure/core/response.hpp"
-#include "azure/core/strings.hpp"
```

Comment:
Why this isn't needed anymore?

Reply:
Was used for Metadata, for `ToLower`. But not longer needed after metadata change.